distribution/docs/src/main/resources/_contents/_configuring/configuring-from-configuration-files-contents.adoc

```diff
@@ -0,0 +1,746 @@
+
+As most configurations are stored in configuration files, in some instances it may make sense to edit those configuration files directly.
+
+==== Configuring ${branding} Global Settings
+
+Global configuration settings are configured via the properties file `system.properties`.
+These properties can be manually set by editing this file or set via the initial configuration from the ${admin-console}.
+
+.Global Settings
+[cols="2,3m,2,3,3m,1"]
+|===
+|Title
```

Comment:
The headers aren't bold when the table is generated, and the `Property` and `Default Value` are monospaced.